addons/website_slides/controllers/main.py

```diff
@@ -301,17 +301,39 @@ def create_slide(self, *args, **post):
     # --------------------------------------------------
     @http.route('/slides/embed/<int:slide_id>', type='http', auth='public', website=True, sitemap=False)
     def slides_embed(self, slide_id, page="1", **kw):
+        """
+        Detect if the slide is embedded in a website outside of the Odoo database.
+        The detection is there to differentiate cases where the slide is present on a webpage that is different than
+        the website for which the slide was created
+
+        This is how we decide if the slide is embedded or not:
+        if referer_url is Falsly -> Not embedded (it's the default behavior)
+
+        if there is no website_id (which means the slide is available for all websites) ->
+            is_embedded = domain of referer is equal to ANY of the domain of the odoo DB
+        else (There is a website_id, so only availabled for that website) ->
+            is_embedded = domain of referer is equal to domain of website
```

Comment:
This description seems to reproduce the code incorrectly? It states that `is_embedded` if `any(referer_domain == website.domain for website in db)`, but the code says the exact opposite.